ci-operator/config/rh-messaging-qe/claire/rh-messaging-qe-claire-lpt__amq-broker-ocp4.17-lp-interop.yaml

```diff
@@ -25,6 +25,7 @@ tests:
     cluster_profile: aws-cspi-qe
     env:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIPS_ENABLED: "true"
```

Comment:
FIPS_ENABLED is added to the wrong test, it should be added to `amq-broker-interop-aws-fips`